### PR TITLE
list comments in frontend and minor changes in backend

### PR DESCRIPTION
**Pull request by daymos**

### CODE REVIEW

File: handler/createTwtt.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-  'use strict'
+'use strict'
   module.exports = controller =>
     (req, res) => {
       let payload = '';
       res.setHeader('content-type', 'application/json');
-      res.writeHead(201);
+      res.writeHead(200);
```
Comment (gabrielperales) [inline handler/createTwtt.js]: [Status code 201](https://httpstatuses.com/201)

File: server.js [hunk 2]
```diff
@@ -19,8 +19,8 @@ const staticFilesHandler = require('./handler/staticFiles');
 
 const r = router([{
   method: 'GET',
-  routeRegExp: '/',
```
Comment (gabrielperales) [inline server.js]: who pushed this?, this morning when @daymos showed me the code it wasn't there, and now we are changing it again?

File: handler/createTwtt.js (hunk 1 above)
Comment (gabrielperales) [inline handler/createTwtt.js]: I thin the code 201 is better when you are **creating** a new twett

Comment (daymos) [inline handler/createTwtt.js]: you are right, I will create a different callback for this code tomorrow. Now I needed this to be 200 to work with the current code.

**Pull request merged**